### ISSUE TITLE
UWP Master Detail menu always expanded and cannot collapse

### ISSUE THREAD

**Issue by Surfrat** [rid00z/FreshMvvm#231]
I have researched this issue and have not found a solution.

This is the menu of my UWP app:
![image](https://user-images.githubusercontent.com/2066487/41584178-e9d1665c-73a6-11e8-863b-541534710ba3.png)

Here is the Android app (iOS is the same):
![image](https://user-images.githubusercontent.com/2066487/41584412-7e19c9bc-73a7-11e8-91ad-58c1b7d172d0.png)

and 
![image](https://user-images.githubusercontent.com/2066487/41584440-8e6a49a4-73a7-11e8-98bc-ac785204163e.png)

Here is my code:
`public partial class App : Application
    {

        public App()
        {
            InitializeComponent();
            Log.Info("App", "--------------------------------------------------------------");
            Log.Info("App", "Starting MiPlanMobile...");
            SetupIoc();
            FreshPageModelResolver.PageModelMapper = new ModelMapper();
            Plugin.Iconize.Iconize.With(new Plugin.Iconize.Fonts.FontAwesomeModule());
            SetupNavigation();
        }

		....

		private void SetupNavigation()
        {
            FreshMasterDetailNavigationContainer masterDetail = new FreshMasterDetailNavigationContainer();
            masterDetail.AddPage<IllustrationListViewModel>("Illustrations");
            masterDetail.AddPage<ProfileViewModel>("Profile");
            masterDetail.AddPage<SettingsViewModel>("Settings");
            masterDetail.Init("Menu");
            MainPage = masterDetail;
            Log.Info("SetupNavigation", "SetupNavigation complete.");
        }		`

How do I get the UWP menu to show the same behaviour as Android/iOS?

**Comment by adibradfield** [comment]
I agree that this is an issue. That definitely isn't the UWP specific way to do a master detail view. Here are 3 Microsoft UWP apps with their menu expanded:
![image](https://user-images.githubusercontent.com/12462105/42309874-d0eefb58-8031-11e8-875b-10cfe00b4541.png)

I'm busy at the moment, but I might have a look and see if I can figure it out over the weekend

**Comment by EmilAlipiev** [comment]
download source code change FreshMasterDetailNavigationContainer.cs  or create your own class and use it instead freshmvvm class. this class is totally independent than Freshmvvm implementation. 
Change the init function of class as below. I am using it and it works fine.
```

  public void Init(string menuTitle, string menuIcon = null)
        {
            MasterBehavior = Xamarin.Forms.Device.Idiom == TargetIdiom.Desktop ? MasterBehavior.Default : Xamarin.Forms.Device.Idiom == TargetIdiom.TV ? MasterBehavior.Split : MasterBehavior.Popover;
            CreateMenuPage(menuTitle, menuIcon);
            RegisterNavigation();
        }
```

**Comment by Surfrat** [comment]
Thank you EmilAlipiev for your fix.

Mine looks slightly different:
` public class CustomFreshMasterDetailNavigationContainer : FreshMasterDetailNavigationContainer
    {
        public void Init(string menuTitle, string menuIcon = null)
        {
            MasterBehavior = MasterBehavior.Popover;
            CreateMenuPage(menuTitle, menuIcon);
            RegisterNavigation();
        }
    }`

Works for UWP, Android and UWP.

**Comment by EmilAlipiev** [comment]
can you please close the issue if it works
